### DISCUSSION TITLE
Notas de las entregas 1 y 2

### DISCUSSION THREAD

**Discussion by santiagolorduy1306** [rramosp/ai4eng.v1#195] [00 - General]
profesor cuando podremos conocer las notas de las dos primeras entregas que suman el 10%

**Comment by Felipe-RA** [comment]
Saludos santiago

Para el Lunes de la proxima semana deberíamos tenerlas.

Igual, cualquier novedad les estamos informando!

**Reply by miguelangelrestrepo** [reply]
Profe, buen día, quería preguntarle a que se debe la penalización de 2.5 en la  primera entrega, gracias y feliz  día.

**Reply by Felipe-RA** [reply]
Saludos miguel

El motivo de la penalización es el contenido del README, o especificamente la falta de este:

![image](https://user-images.githubusercontent.com/49454068/187735362-c8d0ef23-7741-4531-9d85-52ea8d053426.png)

**Reply by miguelangelrestrepo** [reply]
Profe pero en mi repositorio sí está el README

**Reply by Felipe-RA** [reply]
Sí... eso veo.

Ya la tienes tu corregida, la penalización se mantiene para tu compañero

**Reply by Juansebastianrs12** [reply]
Profe buenos días, en las notas de los proyectos tengo una nota de 0, yo realice ambas entregas completas junto con mi compañero, quien tiene una nota de 5, por favor revisar, muchas gracias
att. Juan Sebastian Rodriguez Solarte

**Reply by Felipe-RA** [reply]
Saludos Juan

Ya está corregido, gracias por avisarnos.

Recuerda refrescar los grades cuando entres a la plataforma para ver tu 5

**Comment by davidc-garcia** [comment]
Hola, buen día profe, 
He recibido un correo de que ya deberían estar montadas todas las notas de las entregas, sin embargo veo que los míos aún no se califican o la nota es cero (0). 
Me podrían corregir o informar de cualquier problema?
Gracias, saludos.

**Reply by Felipe-RA** [reply]
Saludos davidc

Necesito tu correo o apellidos para poder echarte una mano

Gracias

**Reply by davidc-garcia** [reply]
Saludos,

davidc.garcia@udea.edu.co

Gracias.

**Reply by Felipe-RA** [reply]
Saludos davidc

Tus notas ya estaban, sólo faltaba que le dieras a refresh grades, ya lo hice por tí no pasa nada.
![image](https://user-images.githubusercontent.com/49454068/188256613-1f1a3062-9133-4ef0-a4b8-ae231df15172.png)

Cualquier otra duda nos comentas.
